docusaurus/docs/recipes/video-player.md

```diff
@@ -0,0 +1,99 @@
+---
+id: video-player
+title: Adding a video player
+sidebar_label: Video player
+---
+
+It's often normal to have the necessity of having videos in your website. In this recipe we will show you how to use [`react-player`](https://github.com/CookPete/react-player) and some common use cases.
+
+## Walk-through
+
+### 1. Install `react-player`
+
+```sh
+npm i react-player
+```
+
+### 2. Use `ReactPlayer` in your project
+
+You just need to import `ReactPlayer` and start using it.
+
+```js
+// ...
+import ReactPlayer from 'react-player';
+
+const Home = () => (
+    <div className={ styles.home }>
+        <ReactPlayer url="https://www.video-url.com" />
+    </div>
+);
+```
+
+### 3. Usage
+
+Now let's go through some common use cases that might be helpful.
+
+> ⚠️ By default `react-player` sets a width of 640px and a height of 360px to the player. You should change it in order to fit your needs.
```

Comment:
If this warning is necessary, it should also be noted that height and width are changed through props (`height` and `width`).